nuntium/templates/nuntium/message/post_submission.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="jumbotron">
+    <div class="container">
+        <h1>{% trans "Your message has been created!" %}</h1>
+        {% if messages %}
```

Comment:
Yep you're right, that's true. I think it could be a good idea to move this logic https://github.com/ciudadanointeligente/write-it/blob/198-add-page-post-submission/nuntium/views.py#L72-L75 over to the template rather than using the Django Messaging Framework